### ISSUE TITLE
Does not work with HyperFido HyperSecu key

### ISSUE THREAD

**Issue by johnrs** [prefiks/u2f4moz#13]
It's working fine with my Yubikey Neo, but doesn't work with my HyperFido HyperSecu key.

I tested using the https://demo.yubico.com/u2f site.

**Comment by prefiks** [comment]
Would you be able to test it with version 0.0.3? It fixes issue with some other device, so hopefully it will help with yours too.

**Comment by johnrs** [comment]
Hello Pawel.

I just grabbed v0.0.3.  My YubiKey Neo works fine.  My HyperFido HyperSecu key fails, but in a different way.  It used to fail immediately.  Now it displays the message to press the blinking button on the key and it waits.  But the LED on the key isn’t blinking.  I pressed the button anyway but nothing happens.

John

```
John Souvestre - New Orleans LA
```

**Comment by johnrs** [comment]
P.S.

After I quit Firefox, I got an error message about it aborting.  And now Chrome won’t see the key at all, so the driver must be hosed.

John

```
John Souvestre - New Orleans LA
```

From: John Souvestre [mailto:John@Souvestre.com] 
Sent: 2015 October 26, Mon 07:10
To: 'prefiks/u2f4moz'
Subject: RE: [u2f4moz] Does not work with HyperFido HyperSecu key (#13)

Hello Pawel.

I just grabbed v0.0.3.  My YubiKey Neo works fine.  My HyperFido HyperSecu key fails, but in a different way.  It used to fail immediately.  Now it displays the message to press the blinking button on the key and it waits.  But the LED on the key isn’t blinking.  I pressed the button anyway but nothing happens.

John

```
John Souvestre - New Orleans LA
```

**Comment by prefiks** [comment]
It may be also problem with stuck program that is talking to usb device, it's names u2f.exe (or u2f if you are using mac or linux), and you should be able to terminate it from task manger (or with killall command in macos or linux).

**Comment by johnrs** [comment]
Yes, I see u2f.exe running.  I killed it.  But my YubiKey still isn’t working right.  Last time I inserted it Windows wanted to install a driver for it, and failed.

I’m going to reboot.

John

```
John Souvestre - New Orleans LA
```

**Comment by johnrs** [comment]
After rebooting everything seems back to normal.

John

```
John Souvestre - New Orleans LA
```

From: John Souvestre [mailto:John@Souvestre.com] 
Sent: 2015 October 26, Mon 07:21
To: 'prefiks/u2f4moz'
Subject: RE: [u2f4moz] Does not work with HyperFido HyperSecu key (#13)

Yes, I see u2f.exe running.  I killed it.  But my YubiKey still isn’t working right.  Last time I inserted it Windows wanted to install a driver for it, and failed.

I’m going to reboot.

John

```
John Souvestre - New Orleans LA
```
